com.riverbankcomputing.PyQt.BaseApp.yaml

```diff
@@ -0,0 +1,96 @@
+app-id: com.riverbankcomputing.PyQt.BaseApp
+runtime: org.kde.Platform
+runtime-version: "6.2"
+sdk: org.kde.Sdk
+command: /usr/bin/python

```

Comment:
Command is NEVER an aboslute path

Reply:
This is a BaseApp and never meant to run directly. This is just a placeholder.

Reply:
I think `command` field is optional though. You can just remove it.